packages/core/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,152 @@
+import { Component, h, Prop, Watch, Element } from "@stencil/core";
+import clsx from "clsx";
+import { tabbable } from "tabbable";
+import { v4 as uuidv4 } from "uuid";
+
+@Component({
+  tag: "dso-dropdown-menu",
+  shadow: true,
+})
+export class DropdownMenu {
+  /**
+   * Whether the menu is open or closed.
+   * This attribute is reflected and mutable.
+   */
+  @Prop({ reflect: true, mutable: true })
+  open = false;
+
+  /**
+   * Alignment of the dropdown
+   */
+  @Prop()
+  dropdownAlign: "left" | "right" = "left";
+
+  @Element()
+  el: HTMLElement | undefined;
```

Comment:
Ik zie nu pas wat je doet, dit had ik bij de vorige review ook al moeten meegeven:

```
@Element()
host!: HTMLElement;
```

Reply:
Oke, maar het gebruik hiervan staat gelijk aan het gebruik van `this.el?`

Reply:
Ja, minus de getter boilerplace

Reply:
Aangepast